ccmlib/scylla_node.py

```diff
@@ -1455,6 +1455,10 @@ def dump_sstables(self,
         :return: return all the partitions collected in the specified sstables
         :raises: subprocess.CalledProcessError if scylla-sstable returns a non-zero exit code.
 
+        only the $PARTITION is returned from this method, please see
```

Comment:
Isn't it the `$SSTABLE` symbol in fact that we return here?
```
$SSTABLE := [$PARTITION, ...]
```
We return multiple partitions.

Reply:
ahh, indeed!